test/integration/image-optimizer/test/index.test.js

```diff
@@ -1108,4 +1108,46 @@ describe('Image Optimizer', () => {
 
     setupTests(true)
   })
+
+  describe('content disposition header', () => {
```

Comment:
I think we can utilize the existing tests and add a check for `content-disposition` just above `content-type` since nearly every test is checking headers

https://github.com/vercel/next.js/blob/2a5dd0b19b0665cfcabde12ad13e9a39096967fd/test/integration/image-optimizer/test/index.test.js#L62

Reply:
yeah, I thought about it at the beginning but noticed that there are not cases for external URLs like in lines 1135-1137

Reply:
There is an external URL test here https://github.com/vercel/next.js/blob/2a5dd0b19b0665cfcabde12ad13e9a39096967fd/test/integration/image-optimizer/test/index.test.js#L355-L356